src/test/java/gov/usds/case_issues/controllers/HitlistApiControllerTest.java

```diff
@@ -176,6 +190,111 @@ public void putCsv_invalidCreationDate_badRequest() throws Exception {
 			0, _uploadService.getUploadHistory(_system, _type).size());
 	}
 
+	@Test
+	@WithMockUser(authorities = "UPDATE_ISSUES")
+	public void putCsv_backDatedWithoutStructureAuthority_forbidden() throws Exception {
+		MockHttpServletRequestBuilder issuePut = putIssues(CSV_CONTENT)
+				.param("effectiveDate", "2019-12-31T20:00:00Z")
+				.content(NO_OP);
+			perform(issuePut).andExpect(status().isForbidden());
+	}
+
+	@Test
+	@WithMockUser(authorities = "UPDATE_STRUCTURE")
+	public void putCsv_backDatedWithoutIssuesAuthority_forbidden() throws Exception {
+		MockHttpServletRequestBuilder issuePut = putIssues(CSV_CONTENT)
+				.param("effectiveDate", "2019-12-31T20:00:00Z")
+				.content(NO_OP);
+			perform(issuePut).andExpect(status().isForbidden());
+	}
+
+	@Test
+	@WithMockUser(authorities = {"UPDATE_ISSUES", "UPDATE_STRUCTURE"})
+	public void putCsv_backDatedIssues_correctDateUsed() throws Exception {
+		String receiptNumber = "FKE27182818";
+		MockHttpServletRequestBuilder issuePut = putIssues(CSV_CONTENT, "2017-05-15T20:00:00Z")
+			.content(CSV_HEADER_SHORT + receiptNumber + ",2001-08-29T00:00:00-04:00,Pay Per View\n");
+		perform(issuePut).andExpect(status().isAccepted());
+		CaseIssueUpload lastUpload = _uploadService.getLastUpload(_system, _type, VALID_ISSUE_TYPE);
+		assertEquals(2017, lastUpload.getEffectiveDate().getYear());
```

Comment:
This is clunky but informative; the alternative is smooth to read and write but doesn’t tell you how the failure happened. 😕